integration_tests/integration_test.go

```diff
@@ -125,21 +125,24 @@ func TestUpdateSpinsvcStatus(t *testing.T) {
 
 	sc := func() error {
 		v := RunCommandAndAssert(fmt.Sprintf("%s -n %s get spinsvc %s -o=jsonpath='{.status.status}'", e.KubectlPrefix(), ns, SpinServiceName), t)
-		if t.Failed() || !assert.Equal(t, spinnakerservice.Ok, strings.TrimSpace(v)) {
```

Comment:
We need this to not rely on the first status check because when the statement is not fulfilled with the assert function an error is thrown.